src/CattyTests/DataModel/SoundTest.swift

```diff
@@ -32,9 +32,52 @@ final class SoundTest: XCTestCase {
         project.scene = scene
         scene.project = project
 
-        let sound = Sound(name: "testLook", fileName: "testSoundFile")
+        let sound = Sound(name: "testSound", fileName: "testSoundFile")
 
         let expectedPath = project.projectPath() + "testScene/sounds/testSoundFile"
         XCTAssertEqual(expectedPath, sound.path(for: scene))
     }
+
+    func testIsEqualToSound() {
+           let project = Project()
+           let scene = Scene(name: "testScene")
+           project.scene = scene
+           scene.project = project
+
+          let sound = Sound(name: "testSound", fileName: "testSoundFile")
+          let equalSound = Sound(name: "testSound", fileName: "testSoundFile")
+
+           XCTAssertTrue(sound.isEqual(to: equalSound))
```

Comment:
Please also add an assert to verify that two sounds are not equal (`XCTAssertFalse`).